test/gtest/stringify.cpp

```diff
@@ -29,85 +29,52 @@ TEST_F(StringifyTest, Print) {
           (i32.const 30)
         ))
       )
-      (block $block_d
```

Comment:
Please undelete this since you support try-catch after all.

Reply:
done